syntax_and_semantics/literals.md

```diff
@@ -20,3 +20,4 @@ Crystal provides several literals for creating values of some basic types.
 | [Tuple](./literals/tuple.html)            | `{1, "hello", 'x'}`                         |
 | [NamedTuple](./literals/named_tuple.html) | `{name: "Crystal", year: 2011}`, `{"this is a key": 1}`|
 | [Proc](./literals/proc.html)              | `->(x : Int32, y : Int32) { x + y }`        |
+| [Command](./literals/command.html) | `` `echo foo` `` |
```

Comment:
Could you please adjust the padding that the `|` character is on the same column as in the previous lines?

Reply:
I’m waiting #346 to be merged to adjust the proper padding.

Reply:
#346 merged

Reply:
I kind of like that literal commands can work with interpolation, as shown in #347.